src/ccsr/src/client.rs

```diff
@@ -293,7 +321,7 @@ impl fmt::Display for PublishError {
                 f,
                 "schema being registered is incompatible with an earlier schema"
             ),
-            PublishError::InvalidSchema => write!(f, "input schema is an invalid avro schema"),
+            PublishError::InvalidSchema { message } => write!(f, "schema: {}", message),
```

Comment:
What do the error messages that come back from the CSR look like? Is "schema:" the right prefix? (Vs either "invalid schema: " or no prefix at all, if the error messages are sufficiently descriptive.)

Reply:
The potential errors all seem to use the word `schema`! Like `"Either the input schema or one its references is invalid"`. I'll remove the `schema:` prefix for now.